README.md

```diff
@@ -40,11 +40,13 @@ addon | version | summary
 [partner_create_by_vat](partner_create_by_vat/) | 10.0.1.0.0 | Using VIES webservice, name and address information will be fetched and added to the partner.
 [partner_employee_quantity](partner_employee_quantity/) | 10.0.1.0.0 | Know how many employees a partner has
 [partner_external_map](partner_external_map/) | 10.0.1.0.0 | Add Map and Map Routing buttons on partner form to open GMaps, OSM, Bing and others
+[partner_financial_risk](partner_financial_risk/) | 10.0.1.0.0 | Manage partner risk
```

Comment:
This shouldn't be changed here. It's changed automatically. Please remove any trace of this or it will conflict as soon as other module is merged in the repository.